src/app/[locale]/global.css

```diff
@@ -24,16 +24,26 @@ img {
 
 a {
   text-decoration: none;
-  display: block;
+  color: black;
+  cursor: pointer;
 }
 
 /* End Normalize */
 
 .slick-dots {
   top: 10rem;
   transform: scale(2);
-  filter: invert(1);
+  /* filter: invert(1); */
```

Comment:
https://www.notion.so/Git-rules-849f8ca8b3db4ded926d762c6fff8798 1.1.10

Reply:
Already fixed in next PR